KalturaWowzaServer/src/main/java/com/kaltura/media_server/services/DiagnosticsProvider.java

```diff
@@ -78,6 +88,29 @@ public void execute(HashMap<String,Object> data, HashMap<String,String > quarySt
             data.put("dateStarted", dateStarted);
             data.put("timeRunning", timeRunning);
             data.put("hostName", hostName);
+            data.put("CpuUsage", CpuUsage);
```

Comment:
1.should be camel case (small C) - at the end it's serialize to Json
2. the value should be a double value and not a string (otherwise hard to parse)